<launch>
  <arg name="norm_control_switch" default="0" />
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node" output="screen">
    <param name="port" value="/dev/ttyUSB1" />
    <param name="baud" value="57600" />
  </node>
  <node pkg="hugmy" type="guidance_manual_controller" name="guidance_manual_controller" output="screen">
    <param name="norm_control_switch" value="$(arg norm_control_switch)" />
  </node>
  <!-- <node pkg="joy" type="joy_node" name="joy_node" output="screen" /> -->

</launch>
